### PR TITLE
#161967080 Setup an empty react project

### PR DESCRIPTION
**Pull request by Bryan-Cee**
#### What does this PR do?

- Set up React project with Airbnb linting option

#### Description of Task to be completed?

- Create the initial boilerplate
- Update readme
- Add eslint to the project

#### How should this be manually tested?

- Clone this repo from the terminal

```
git clone https://github.com/andela/ah-code-titans-frontend.git
```

- Then cd into the projects folder and checkout to the specific branch
```
cd ah-code-titans-frontend
git checkout ch-create-react-app-161967080
```
- Install all dependencies
```
npm install
```
- Run the app
```
npm start
```
#### What are the relevant pivotal tracker stories?
[#161967080](https://www.pivotaltracker.com/story/show/161967080)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+```
+- node
+- browser preferbly (chrome)
```
Comment (jillo-abdullahi) [inline README.md]: 'Preferably' misspelled here. Please rectify.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "not op_mini all"
+  ],
+  "devDependencies": {
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.12.3",
+    "undefined": "^0.1.0"
+  }
+}
```
Comment (jillo-abdullahi) [inline package.json]: Please add a new line here.

**Review by jillo-abdullahi — CHANGES_REQUESTED** [review]
We won't be using Service Worker, so I think you can do away with the `manifest.json` file.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    />
+    <meta name="theme-color" content="#000000" />
+    <!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
```
Comment (denomash) [inline public/index.html]: What is `<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />` for ?

**Review by denomash — APPROVED** [review]

File: public/index.html (hunk 3 above)
Comment (Bryan-Cee) [reply public/index.html]: It was for the manifest file let me remove it to

**Comment by denomash** [comment]
👍

**Review by bettblake08 — APPROVED** [review]
👍

**Pull request merged**